.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2020": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+};
```

Comment:
**NOTE:** Microverse has moved on to Github actions for continuous deployment please use that instead of stickler. 
![image](https://user-images.githubusercontent.com/31897434/85197008-14353c00-b2d6-11ea-932b-d3a0ab291c2a.png)

https://github.com/microverseinc/linters-config/tree/master/javascript


Remember not to make any changes to the linter's files, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
You need to fix linter's error.
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/